cpp/new_journal_alert.cc

```diff
@@ -131,6 +132,33 @@ bool NewIssueInfo::operator<(const NewIssueInfo &rhs) const {
 }
 
 
+bool NewIssueInfo::operator==(const NewIssueInfo &rhs) const {
+    return control_number_ == rhs.control_number_ and
```

Comment:
Wouldn't this line suffice?

Reply:
Probably mostly yes but might lead to sublte errors.